.github/workflows/build-lint-test.yml

```diff
@@ -0,0 +1,149 @@
+name: Build, lint and test
+
+on:
+    # Triggers the workflow on push or pull request events but only for the main branch and release branches
+    push:
+        branches: [main, release**]
+    pull_request:
+        branches: [main, release**]
+
+    # Allows us to run the workflow manually from the Actions tab
+    workflow_dispatch:
+
+jobs:
+    dependencies:
+        runs-on: ubuntu-20.04
+
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Cache dependencies
+              id: fetch-dependencies
+              uses: actions/cache@v2
+              with:
+                  path: |
+                      **/node_modules
+                  key: ${{ runner.os }}-v3-${{ hashFiles('**/yarn.lock') }}
+
+            - uses: actions/setup-node@v2
+              if: steps.fetch-dependencies.outputs.cache-hit != 'true'
+              with:
+                  node-version: '14.16.0'
+
+            - name: Download dependencies
+              if: steps.fetch-dependencies.outputs.cache-hit != 'true'
+              run: yarn
+
+    build:
+        runs-on: ubuntu-20.04
+        needs: [dependencies]
+
+        steps:
+            - uses: actions/checkout@v2
+              with:
+                  submodules: 'recursive'
+            - uses: actions/setup-node@v2
+              with:
+                  node-version: '14.16.0'
+
+            - name: Get cached dependencies
+              uses: actions/cache@v2
+              with:
+                  path: |
+                      **/node_modules
+                  key: ${{ runner.os }}-v3-${{ hashFiles('**/yarn.lock') }}
+
+            - name: Build
+              run: yarn build:dev
```

Comment:
All these commands are for the browser wallet as far as I see. What about checking the other packages and examples to cover all the code included in the repository? Or am I missing some `yarn workspace` magic stuff?

Reply:
Yeah, I guess it only builds the wallet itself.
But the message hub is used by the wallet, and is the api not also?
But you are correct, I'll try adding it, so that it builds the example as well.

Reply:
And type check.